dotcom-rendering/src/components/VideoAtom.stories.tsx

```diff
@@ -1,76 +1,58 @@
 import { css } from '@emotion/react';
+import type { Decorator, Meta, StoryObj } from '@storybook/react';
 import { VideoAtom } from './VideoAtom';
 
-export default {
-	title: 'VideoAtom',
+const meta = {
+	title: 'Components/VideoAtom',
 	component: VideoAtom,
 	parameters: {
 		// Chromatic ignores video elements by design so there's no point trying to snapshot here
 		// https://www.chromatic.com/docs/ignoring-elements
 		chromatic: { disable: true },
 	},
-};
+} satisfies Meta<typeof VideoAtom>;
 
-export const DefaultStory = () => {
-	return (
-		<div
-			css={css`
-				width: 800px;
-				margin: 25px;
-			`}
-		>
-			<VideoAtom
-				poster="https://media.guim.co.uk/29638c3179baea589b10fbd4dbbc223ea77027ae/0_0_3589_2018/master/3589.jpg"
-				assets={[
-					{
-						url: 'https://uploads.guim.co.uk/2020%2F23%2F04%2Ffor+testing+purposes+only--ef8e62ab-bc06-4892-8da1-65a7e5bacb77-1.mp4',
-						mimeType: 'video/mp4',
-					},
-				]}
-			/>
-		</div>
-	);
-};
+export default meta;
 
-export const LargeStory = () => {
-	return (
-		<div
-			css={css`
-				width: 800px;
-				margin: 25px;
-			`}
-		>
-			<VideoAtom
-				poster="https://media.guim.co.uk/29638c3179baea589b10fbd4dbbc223ea77027ae/0_0_3589_2018/master/3589.jpg"
-				assets={[
-					{
-						url: 'https://uploads.guim.co.uk/2020%2F23%2F04%2Ffor+testing+purposes+only--ef8e62ab-bc06-4892-8da1-65a7e5bacb77-1.mp4',
-						mimeType: 'video/mp4',
-					},
-				]}
-				height={500}
-				width={880}
-			/>
-		</div>
-	);
-};
+type Story = StoryObj<typeof meta>;
 
-export const NoPosterStory = () => {
-	return (
-		<div
-			css={css`
-				width: 800px;
-				margin: 25px;
-			`}
-		>
-			<VideoAtom
-				assets={[
-					{
-						url: 'https://uploads.guim.co.uk/2020%2F23%2F04%2Ffor+testing+purposes+only--ef8e62ab-bc06-4892-8da1-65a7e5bacb77-1.mp4',
-						mimeType: 'video/mp4',
-					},
-				]}
-			/>
-		</div>
-	);
-};
+const sizeDecorator: Decorator = (Story) => (
+	<div
+		css={css`
+			width: 800px;
+			margin: 25px;
+		`}
+	>
+		<Story />
+	</div>
+);
```

Comment:
Is this too early an abstraction seeing it's only used once? We could easily inline it in the component.

Reply:
It's only used in the stories I believe, rather than the component? I saw it was written out three times, so I pulled it into a function. Technically, with composable stories, the function is only referenced once in the first story, but it is used in all three.

Reply:
I see – looking in the final file it only appears once, so it could be inlined in the `Default` component’s `decorators`, as it’s not referenced anywhere else. [Much clearer without whitespace](https://github.com/guardian/dotcom-rendering/pull/11161/files?diff=split&w=1).